app/models/ScmemeVariance.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json._
+import utils.{Enumerable, WithName}
+
+import scala.language.implicitConversions
+
+
+case class SchemeVariance(psaId: String, srn: String)
+
+object SchemeVariance {
+  implicit val format: OFormat[SchemeVariance] = Json.format[SchemeVariance]
+}
+
+sealed trait Lock
+
+case object VarianceLock extends WithName("SuccessfulVarianceLock") with Lock
```

Comment:
Better name might be something like "VarianceLockSuccessful"

Reply:
fixed